tests/conftest.py

```diff
@@ -1,117 +1,57 @@
-def strategist(accounts):
-    # You! Our new Strategist!
+def management(accounts):
     yield accounts[3]
 
 
 @pytest.fixture
-def keeper(accounts):
-    # This is our trusty bot!
+def strategist(accounts):
     yield accounts[4]
 
 
 @pytest.fixture
-def strategy(strategist, keeper, vault, Strategy):
-    strategy = strategist.deploy(Strategy, vault)
-    strategy.setKeeper(keeper)
-    yield strategy
-
-
-@pytest.fixture
-def nocoiner(accounts):
-    # Has no tokens (DeFi is a ponzi scheme!)
+def keeper(accounts):
     yield accounts[5]
 
 
 @pytest.fixture
-def pleb(accounts, andre, token, vault):
-    # Small fish in a big pond
-    a = accounts[6]
-    # Has 0.01% of tokens (heard about this new DeFi thing!)
-    bal = token.totalSupply() // 10000
-    token.transfer(a, bal, {"from": andre})
-    # Unlimited Approvals
-    token.approve(vault, 2 ** 256 - 1, {"from": a})
-    # Deposit half their stack
-    vault.deposit(bal // 2, {"from": a})
-    yield a
+def token():
+    token_address = "0x6b175474e89094c44da98b954eedeac495271d0f"  # this should be the address of the ERC-20 used by the strategy/vault
+    yield Contract.from_explorer(token_address)
 
 
 @pytest.fixture
-def chad(accounts, andre, token, vault):
-    # Just here to have fun!
-    a = accounts[7]
-    # Has 0.1% of tokens (somehow makes money trying every new thing)
-    bal = token.totalSupply() // 1000
-    token.transfer(a, bal, {"from": andre})
-    # Unlimited Approvals
-    token.approve(vault, 2 ** 256 - 1, {"from": a})
-    # Deposit half their stack
-    vault.deposit(bal // 2, {"from": a})
-    yield a
-
-
-@pytest.fixture
-def greyhat(accounts, andre, token, vault):
-    # Chaotic evil, will eat you alive
-    a = accounts[8]
-    # Has 1% of tokens (earned them the *hard way*)
-    bal = token.totalSupply() // 100
-    token.transfer(a, bal, {"from": andre})
-    # Unlimited Approvals
-    token.approve(vault, 2 ** 256 - 1, {"from": a})
-    # Deposit half their stack
-    vault.deposit(bal // 2, {"from": a})
-    yield a
+def vault(pm, gov, rewards, guardian, management, token):
+    Vault = pm(config["dependencies"][0]).Vault
+    vault = guardian.deploy(Vault)
+    vault.initialize(token, gov, rewards, "", "", guardian)
+    vault.setDepositLimit(2 ** 256 - 1, {"from": gov})
+    vault.setManagement(management, {"from": gov})
+    yield vault
 
 
 @pytest.fixture
-def whale(accounts, andre, token, vault):
-    # Totally in it for the tech
-    a = accounts[9]
-    # Has 10% of tokens (was in the ICO)
-    bal = token.totalSupply() // 10
-    token.transfer(a, bal, {"from": andre})
-    # Unlimited Approvals
-    token.approve(vault, 2 ** 256 - 1, {"from": a})
-    # Deposit half their stack
-    vault.deposit(bal // 2, {"from": a})
-    yield a
+def strategy(strategist, keeper, vault, Strategy, gov):
+    strategy = strategist.deploy(Strategy, vault)
+    strategy.setKeeper(keeper)
+    vault.addStrategy(strategy, 10_000, 10_000, 10, {"from": gov})
```

Comment:
I would prefer:
`vault.addStrategy(strategy, 10_000, 0, 1_000, {"from": gov})`

Reply:
ok, could you explain why?

Reply:
rate = 0 is unlimited.